### PR TITLE
[IMP] edi_oca: Show traceback on exception errors

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Add ability to show traceback when exceptions are raised

CC @ForgeFlow @etobella

### CODE REVIEW

File: edi_oca/tests/test_backend_process.py [hunk 1]
```diff
@@ -58,7 +58,6 @@ def test_process_record_with_error(self):
             [
                 {
                     "edi_exchange_state": "input_processed_error",
-                    "exchange_error": "OOPS! Something went wrong :(",
```
Comment (etobella) [inline edi_oca/tests/test_backend_process.py:61]: Can you try to verify if te message is inside it :smile: ?

**Review by etobella — APPROVED** [review]

File: edi_oca/tests/test_backend_process.py (hunk 1 above)
Comment (HviorForgeFlow) [reply edi_oca/tests/test_backend_process.py:61]: Sure!

**Review by JaumeBforgeFlow — APPROVED** [review]

**Comment by simahawk** [comment]
@HviorForgeFlow cool! Can you please have a look at #855. We need to return the results to the job too.

**Review by simahawk — APPROVED** [review]

**Comment by etobella** [comment]
@HviorForgeFlow Can you port it to 15 and 16?

**Pull request merged**